tests/tensor_search/test_delete_documents.py

```diff
@@ -1,271 +1,271 @@
 import datetime
 import unittest
-from copy import deepcopy
+from copy import deepcopy, copy
 import marqo.tensor_search.delete_docs
 from marqo.tensor_search.models.delete_docs_objects import MqDeleteDocsRequest, MqDeleteDocsResponse
 from marqo.tensor_search import delete_docs
 import marqo.tensor_search.tensor_search
+from marqo.core.index_management import index_management
+from marqo.core.models.marqo_index import Model
 from marqo.tensor_search import tensor_search
-from marqo.api.exceptions import IndexNotFoundError
 from tests.marqo_test import MarqoTestCase
 import requests
 from unittest.mock import patch
-from marqo.api import exceptions
+from marqo.core import exceptions as core_exceptions
+from marqo.api import exceptions as api_exceptions
 from marqo.tensor_search import enums
-from tests.utils.transition import add_docs_caller
+from tests.utils.transition import add_docs_caller, add_docs_batched
 import os
+from marqo.vespa.models.delete_document_response import DeleteBatchDocumentResponse, DeleteBatchResponse
 
-@unittest.skip
 class TestDeleteDocuments(MarqoTestCase):
     """module that has tests at the tensor_search level"""
 
-    def setUp(self) -> None:
-        self.endpoint = self.authorized_url
+    @classmethod
+    def setUpClass(cls) -> None:
+        super().setUpClass()
 
-        self.generic_header = {"Content-type": "application/json"}
-        self.index_name_1 = "my-test-index-1"
+        text_index_with_random_model = cls.unstructured_marqo_index_request(model=Model(name='random'))
```

Comment:
To be clear, this is not an index. It's an index request

Reply:
nice catch, updated variable name to be more clear!